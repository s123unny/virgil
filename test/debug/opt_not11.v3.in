b ../core/opt_not11.v3 3
b ../core/opt_not11.v3 4
b ../core/opt_not11.v3 5
run
bt
info l
c
q
