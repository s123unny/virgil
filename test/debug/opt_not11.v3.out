breakpoint|../core/opt_not11.v3 3|0
breakpoint|../core/opt_not11.v3 4|1
breakpoint|../core/opt_not11.v3 5|2
stop|stopOnBreakpoint
bt|main|../core/opt_not11.v3|3
variable|0|a|0|int|false
variable|1|b|0|byte|false
variable|2|c|0|byte|false
stop|end
result|false
